.travis_quickstart.sh

```diff
@@ -36,90 +39,98 @@ cd ${DIST_BIN_DIR}
 # Test quick-start-batch
 bin/quick-start-batch.sh &
```

Comment:
Do you want to use one of the quick-starts for the relative path test?

Reply:
hybrid test is for relative path